misc/packaging/gen_acrn_deb.py

```diff
@@ -90,6 +90,7 @@ def create_acrn_deb(board, scenario, version, build_dir):
         'Package: acrn-hypervisor\n',
         'version: %s \n' % version,
         'Depends: libcjson1\n',
+        'Pre-Depends: libsdl2-dev\n',
```

Comment:
This is incorrect, we need the `libsdl2-2.0-0` package and not the development package. Can you please submit a new PR to correct this?

Reply:
Oh, I'll submit a new PR at once